### PR TITLE
Add Swift Argument Parser 1.3.0

### PR DESCRIPTION
**Pull request by SimplyDanny**
https://github.com/apple/swift-argument-parser/releases/tag/1.3.0

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @keith, @jpsim, modules you maintain (swift_argument_parser) have been updated in this PR. Please review the changes.

**Comment by SimplyDanny** [comment]
I assume [apple/swift-argument-parser](https://github.com/apple/swift-argument-parser) needs to provide a stable source code archive as part of their release first ...

**Comment by keith** [comment]
we can use the label added here to skip that check instead. I think trying to change the apple release process for these libraries is probably too hard to be worth it.

File: modules/swift_argument_parser/1.3.0/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: macos
```
Comment (keith) [inline modules/swift_argument_parser/1.3.0/presubmit.yml:4]:
```suggestion
    platform: macos
    bazel: 7.x
```

looks like a new requirement

**Review by keith — APPROVED** [review]

File: modules/swift_argument_parser/1.3.0/presubmit.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    bazel: 7.x
+    build_targets:
+      - '@swift_argument_parser//:ArgumentParser'
+    build_flags:
+      - --macos_minimum_os=12.0
```
Comment (keith) [inline modules/swift_argument_parser/1.3.0/presubmit.yml:9]:
```suggestion
      - --macos_minimum_os=12.0
      - "--repo_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1"
```

**Pull request merged**